clients/src/main/resources/common/message/FetchResponse.json

```diff
@@ -102,6 +104,15 @@
           "about": "The preferred read replica for the consumer to use on its next fetch request"},
         { "name": "Records", "type": "records", "versions": "0+", "nullableVersions": "0+", "about": "The record data."}
       ]}
+    ]},
```

Comment:
Should we be using the same tag here as diverging epoch?

Reply:
it looks like tags are scoped to the list level so this isn't really the same tag. They also need to be contiguous within their scope so this gives an error if I try to tag NodeEndpoints to something other than 0.